docs/designers-developers/developers/tutorials/block-tutorial/writing-your-first-block-type.md

```diff
@@ -13,13 +13,13 @@ While the block's editor behaviors are implemented in JavaScript, you'll need to
 
 function gutenberg_boilerplate_block() {
 	wp_register_script(
-		'gutenberg-boilerplate-es5-step01',
-		plugins_url( 'step-01/block.js', __FILE__ ),
+		'gutenberg-examples-01-basic',
+		plugins_url( '01-basic/block.js', __FILE__ ),
 		array( 'wp-blocks', 'wp-element' )
 	);
 
-	register_block_type( 'gutenberg-boilerplate-es5/hello-world-step-01', array(
-		'editor_script' => 'gutenberg-boilerplate-es5-step01',
+	register_block_type( 'gutenberg-examples/01-basic', array(
```

Comment:
The gutenberg-examples repo calls the block `example-01-basic`, it does duplicate the example name

```suggestion
	register_block_type( 'gutenberg-examples/example-01-basic', array(
```